infra/modules/load-balancer/main.tf

```diff
@@ -37,15 +37,29 @@ resource "google_compute_backend_bucket" "cdn" {
 
 resource "google_compute_health_check" "cloud_deployment" {
   project = var.project_id
-  name    = "cloud-deployment-gke-golang"
+  name    = "cloud-deployment-gke-golang-health"
   http_health_check {
     port_specification = "USE_SERVING_PORT"
   }
 }
 
+resource "google_compute_firewall" "cloud_deployment" {
+  name      = "cloud-deployment-gke-golang-health-check"
```

Comment:
**Just a heads-up (in case it wasn't intentional):** 
* This `google_compute_firewall` resource is named `...-health-check`.
* And the `google_compute_health_check` is named `...-health`.
No action needed. I thought you might have wanted something like:
```
resource "google_compute_health_check" "cloud_deployment" {
  project = var.project_id
  name    = "cloud-deployment-gke-golang-health-check"
```
and 
```
resource "google_compute_firewall" "cloud_deployment" {
  name      = "cloud-deployment-gke-golang-firewall"
```

Reply:
Yeah, this was just lifted from the deleted module and I didn't pay much attention to the name during debugging the other issues.  I renamed it to `cloud-deployment-gke-golang-allow-health-check` to reflect its purpose.